### DISCUSSION TITLE
v2.7.3 / [预览] v2.8.1

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4296] [分享 / Share]
- 修复 VideoInfo 报错 `Cannot read properties of null (reading 'list')`. (影响 `下载字幕`, `查看封面` 等功能) (#4291)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.7.3'>v2.7.3 / [预览] v2.8.1</a>.</em>

**Comment by peace-max** [comment]
新版本现在出现，外置稍后再看，和快速收藏，都出现bug了。
在list页面工具栏不显示了。
浏览器版本：百分浏览器版本 5.0.1002.354（正式版本） （32 位） (Portable) (Chromium 102.0.5005.167)

**Reply by the1812** [reply]
见 #4301, v2.7.2 也有这个问题, 应该算是另一回事, 新版没动这块

**Reply by kethoven** [reply]
应该是的。没注意到b站也做了改动，以为是脚本的新bug。

**Reply by CRYS74L** [reply]
双击全屏同样失效了

**Comment by kethoven** [comment]
bug反馈
这个版本的稍后再看外置、快速收藏功能失效了。

**Reply by kethoven** [reply]
在傲游浏览器上，所有页面都失效，在一般的视频播放页面也是如此。

**Comment by 919334339** [comment]
预览版2.8.1还是不行
![image](https://github.com/the1812/Bilibili-Evolved/assets/100505639/45edf09e-8b8a-40c2-bffc-de2e054837c8)

**Reply by Geeyun-JY3** [reply]
见 issue 区置顶：https://github.com/the1812/Bilibili-Evolved/issues/4303#issuecomment-1636727701

**Comment by Marx48** [comment]
> 预览版2.8.1还是不行 
这里也不起作用，我使用的是2.8.1preview的, 我的浏览器是Microsoft Edge

![image](https://user-images.githubusercontent.com/100505639/253737504-45edf09e-8b8a-40c2-bffc-de2e054837c8.png)

**Reply by Tinhone** [reply]
可以查看 #4303 来了解这个问题的详细信息
